### PR TITLE
Years of experience and Salary dataset

### PR DESCRIPTION
**Pull request by BhaswatiRoy**
## Issue Title :

- Info about the related issue (Aim of the project)

- [x] SWOC Participant 
- [x] Contributor

Closes: #issue number that will be closed through this PR

### Describe the add-ons or changes you've made

Give a clear description of what have you added or modifications made

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how it has been tested
Describe how have you verified the changes made

## Checklist:

- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added things that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: Years of experience and Salary dataset/Model/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (abhisheks008) [inline Years of experience and Salary dataset/Model/README.md:1]: Follow the README.md template and insert proper description of the project. Follow this for reference, https://github.com/abhisheks008/ML-Crate/blob/main/.github/readme_template.md

File: Years of experience and Salary dataset/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (abhisheks008) [inline Years of experience and Salary dataset/README.md:1]: Delete this file.

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Do the needful changes @BhaswatiRoy

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
Delete the Model/README.md file @BhaswatiRoy

**Comment by BhaswatiRoy** [comment]
I've done the required changes.
Do let me know if anything else is needed!

**Review by abhisheks008 — APPROVED** [review]

**Comment by abhisheks008** [comment]
Good work keep it up @BhaswatiRoy

**Comment by BhaswatiRoy** [comment]
> Good work keep it up @BhaswatiRoy

Thank you

**Pull request merged**